src/MiniValidation/MiniValidator.cs

```diff
@@ -101,6 +139,28 @@ public static bool TryValidate<TTarget>(TTarget target, bool recurse, bool allow
             throw new ArgumentNullException(nameof(target));
         }
 
+        return TryValidate(target, null, recurse, allowAsync, out errors);
```

Comment:
Rather than having the old public methods call the new public methods and introduces multiple layers of argument null checking of the same arguments, refactor the existing methods to have a private `Impl` method that all the public methods call, that doesn't do any argument checking.